test/e2e/gateway/e2e_suite_test.go

```diff
@@ -12,6 +12,7 @@ import (
 var _ = Describe("Test Gateway on Universal", Label("arm-not-supported"), gateway.GatewayOnUniversal)
 var _ = Describe("Test Gateway on Kubernetes", Label("arm-not-supported"), gateway.GatewayOnKubernetes)
 var _ = Describe("Test Gateway on Kubernetes with HELM", Label("arm-not-supported"), gateway.GatewayHELM)
+var _ = Describe("Cross-Mesh Gateway on Kubernetes", Label("arm-not-supported"), gateway.CrossMeshGatewayOnKubernetes)
```

Comment:
can we implement this test in the new `e2e_env/kubernetes`?

Reply:
Sure